src/hal/src/pass.rs

```diff
@@ -35,33 +35,37 @@ pub type AttachmentLayout = image::ImageLayout;
 #[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct AttachmentOps {
-    ///
+    /// Whether or not data from the load operation will be preserved after the subpass.
```

Comment:
That description is a bit confusing. It indicates how the data of the attachment will be load at first usage at the beginning of the subpass.